tests/shellchecks.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Based on https://github.com/koreader/koreader/blob/master/.ci/helper_shellchecks.sh
+
+mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' && git ls-files ./*.sh; } | sort | uniq)
+
+SHELLSCRIPT_ERROR=0
+
+for shellscript in "${shellscript_locations[@]}"; do
+	echo -e "${ANSI_GREEN}Running shellcheck on ${shellscript}"
```

Comment:
Is it needed to restore the original colour at the end of the script and/or in case of error / interruption?

Reply:
You mean for local use?

Reply:
Yes (indeed, if the shell exits, there is no need)